deploy/sql/vso-validate-migrations/vso-migrate-up.sh

```diff
@@ -1,16 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
This is used to [validate all migrations](https://github.com/DFEAGILEDEVOPS/MTC/blob/master/deploy/build-defs/validate-pr-db-migrations.yml) as a gated PR check.  
We should move it to the `deploy/build-defs` directory or add the content from here and make it inline